main.go

```diff
@@ -109,6 +42,12 @@ var serverHelp = `
     of man-in-the-middle attacks (defaults to the CHISEL_KEY environment
     variable, otherwise a new key is generate each run).
 
+    --ciphers, An optional string comma separated to provide a list of ciphers to
+    use by order of preference and matching the client (default %q).
```

Comment:
Could you please switch the `%q` to ` + defaultCiphers() + `,
similarly with `defaultMACs()`
then we don't need the `fmt.Sprintf` - I think doesn't scale well with large strings, as you'll struggle to find all the placeholders then match them up with the ordered arguments.

I think the more scalable solution would be Go text templates. Then we can use named placeholders `{{ .DefaultMACs }}` etc. However for this, I think the string concat is the simplest and most readable.

Reply:
Agreed, I'm ready to make the change for the go template one, seems more scalable to me too, sound good to you ?

Reply:
Sorry for the delay! Fire away with the Go template solution